ACPs/77-reinventing-subnets/README.md

```diff
@@ -564,7 +569,7 @@ The continuous fee mechanism outlined above does not apply to inactive Subnet Va
 
 ### Should they still be called Subnets?
 
-Through this ACP, Subnets have far more sovereignty than they did before. The Avalanche Community could take this opportunity to re-brand Subnets.
+Going forward, Subnets should be referred to as L1s. Unlike layer 2 blockchains, layer 1 blockchains operate independently without relying on other systems for their core functions. They manage their own consensus mechanisms, transaction processing, and security protocols directly within their own networks. Subnets have always operated this way. With the added functionality introduced in this ACP, they are clearly standalone networks, or L1s.
```

Comment:
I think it could be worth mentioning the fact that L1 validator sets creating using the new ACP-77 flow will not be a subset of the primary network validator set, making it technically incorrect to call them Subnets.

This is neither here or there, and I see the note in the abstract also, but feel like this explanation belongs more in "motivation" since it is no longer an open question IMO

Reply:
my original pr included it as the first part of the motivation, but @dhrubabasu disagrees. I've linked it in line 19 but am open to moving it